.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# All contributions, however small are valued!
+
+# Steps to contribute
+
+ If you want to make If it's a small change, go ahead and raise a pull request, otherwise follow these steps:
```

Comment:
Typo … line 5 here starts one sentence, then mid-way starts a different sentence.

Reply:
Thank you for catching that copy pasta error!